RELEASE_NOTES.md

```diff
@@ -6,6 +6,38 @@
 
 Release Procedure is defined in the [CONTRIBUTING](CONTRIBUTING.md#release-procedure) document.
 
+## v4.3.0
+
+Highlights:
+
+- ICA
+
+### Upgrade instructions
+
+- Note: Skip this step if you build binary from source and are able to properly link libwasmvm.
+
+  - Download `libwasmvm`:
+
+  ```bash
+  $ wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/libwasmvm.$(uname -m).so -O /lib/libwasmvm.$(uname -m).so
+  ```
+
+- Download latest binary or build from source.
+- Run the binary to make sure it works for you: `umeed --version`
+- Wait for software upgrade proposal to pass and trigger the chain upgrade.
+- Swap binaries.
+- Restart the chain.
+- Ensure latest Peggo (v1.4.0) is running
+- Ensure latest Price Feeder (v2.1.0) is running and check your price feeder config is up to date.
```

Comment:
Can you change price-feeder to ojo price-feeder (https://github.com/ojo-network/price-feeder/releases/tag/umee%2Fv2.1.1-rc2)

Reply:
I don't think it's stable. They should manage their dependency. Latest stable release is still in our repo.

Reply:
Ok